test/object-store/realm.cpp

```diff
@@ -1005,6 +1005,60 @@ TEST_CASE("SharedRealm: close()") {
     }
 }
 
+TEST_CASE("SharedRealm: delete_files()") {
+    TestFile config;
+    config.schema_version = 1;
+    config.schema = Schema{{"object", {{"value", PropertyType::Int}}}};
+    auto realm = Realm::get_shared_realm(config);
+
+    // We need to create some additional files that might not be present
+    // for a freshly opened realm but need to be tested for as the will
+    // be created during a Realm's life cycle.
+    // .log_a and .log_b are legacy versions of .log that might be present
```

Comment:
this comment would be nice to have at the place where the file list is coming from in core as well

Reply:
Good point! I'll keep this comment open and move it after we've decided where to get the extensions from.

Reply:
Would acddea5 suffice?

Reply:
yes, thanks for that!